group_vars/betabarrel_cluster/ip_addresses.yml

```diff
@@ -1,7 +1,10 @@
 ---
 ip_addresses:
   betabarrel:
-    vlan983:
-      address: 172.23.41.225
+    bb_internal_management:
+      address: 129.125.55.13
```

Comment:
I do not understand this change: 129.125.55.13 is an external interface; not an internal management interface... Also do not understand why vlan983 had to be removed.

Reply:
Fixed in new commits. Thanks.